CONTRIBUTING.md

```diff
@@ -84,6 +84,53 @@ This also sets up gulp and Jekyll to watch for file changes to the `/docs`
 and `/src` directories and rebuilds the website accordingly. The command is
 aliased by `npm start`.
 
+#### On a Windows machine:
+Install Node.js and npm. This page is helpful: http://blog.teamtreehouse.com/install-node-js-npm-windows <br>
```

Comment:
Hey! Can we add clarification to why this page is helpful? (Is it helpful with both installations, or just one? For everyone, or just novices?)